Automattic/jetpack
# Update/wpcomify modules tests

#### Changes proposed in this Pull Request:


* WPCOMify jetpack modules tests to be used in WPOM context 

#### Testing instructions:





* Nothing really, just make sure the tests are still green, and all of them still included in CI
* Check the total number of tests in CI. it should be equal to `1379`

#### Proposed changelog entry for your changes:


* not needed
